### PR TITLE
[DO NOT MERGE] Changes for FOSSGIS 2017

### PR DESCRIPTION
**Pull request by oxidase**
dont merge

### CODE REVIEW

File: debug/index.html [hunk 1]
```diff
@@ -125,7 +125,7 @@
         },
         "osrm": {
             "type": "vector",
-            "tiles" : ["http://router.project-osrm.org/tile/v1/car/tile({x},{y},{z}).mvt"]
+            "tiles" : ["http://localhost:5000/tile/v1/car/tile({x},{y},{z}).mvt"]
```
Comment (yuryleb) [inline debug/index.html:128]: It seems you forgot to discard your local changes :wink:

Comment (oxidase) [reply debug/index.html:128]: that's ok, the PR should not be merged and should be used only for comments. I specially made it to use a local server and will close the PR after the workshop without merging.

**Comment by daniel-j-h** [comment]
@oxidase good to close here?

Should we keep the branch alive and not delete it in case people come across the notes?

**Comment by oxidase** [comment]
Closed, but keeping branch alive for some time

**Pull request closed without merging**